### DISCUSSION TITLE
my show scramble tool doesnt work on cstimer+

### DISCUSSION THREAD

**Discussion by Jayodhya-cubing** [cs0x7f/cstimer#456] [General]
at least it doesnt work for new updates. it works fine up until 2019 update

**Comment by topppits** [comment]
Do you understand English? I'm starting to doubt that you understand any replies you're getting here.

Please try to understand what we are telling you:


#455 
#432

**Comment by Jayodhya-cubing** [comment]
nope i get it

**Comment by Jayodhya-cubing** [comment]
then whos the guy developed the plus
and apologies if I disturbed you
I just wanted to fix it somehow

**Comment by Jayodhya-cubing** [comment]
I saw a new update on csTimer

**Comment by Jayodhya-cubing** [comment]
And I think the issue is fixed

**Comment by Jayodhya-cubing** [comment]
Yh. It said that it is fixed. Thank You guys a lot
